SlackAPI/SlackAPI.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```

Comment:
@gpailler , Correct me if I'm wrong, but updating the Newtonsoft version here would require anyone using the library to also have to update their Newtonsoft to `>=11.0.2`?

Reply:
From what I know yes. It's better to target the lowest version to maximize the compatibility.